### PR TITLE
Test on PHP 7.3 on Travis

### PR DESCRIPTION
**Pull request by sanmai**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ php:
   - 7.0
   - 7.1
   - 7.2
+  - 7.3
```
Comment (peter279k) [inline .travis.yml:10]: The `php-7.4` version is stable now.

I think it should add `php-7.4` version test on Travis CI build.

Comment (sanmai) [reply .travis.yml:10]: Sure. Time flies!

**Comment by sanmai** [comment]
Travis CI is no longer.

**Comment by DQNEO** [comment]
Sorry for the delay. I just forgot I have the permission to this repo.
I don't want to spoil your effort. Let me take this over.

**Comment by DQNEO** [comment]
I tried to apply this, and found that t's not that easy. https://github.com/mercari/dietcube/pull/35

We need more higher view on the overall project management.


Le me close this :bow:

**Pull request closed without merging**